Generators/src/GeneratorFileOrCmd.cxx

```diff
@@ -0,0 +1,207 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \author R+Preghenella - August 2017
```

Comment:
This is probably a copy-paste error!?

Reply:
You mean the `\author` - yes, that's copy'n'paste oversight.  I don't mind if you don't mind.

Reply:
```suggestion
```

Reply:
We should avoid any authorship information beyond the license, actually.